webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -71,6 +76,8 @@ const ExperimentSingleTrialTabs: React.FC<Props> = ({
     modalOpen: openHyperparameterSearchModal,
   } = useModalHyperparameterSearch({ experiment });
 
+  const waitingForTrials = !trialId && !wontHaveTrials;
```

Comment:
I would advise against `wontHaveTrials` the boolean and `WontHaveTrials` the component.   I think it makes sense to have a component for the message since it will show up in a few places, but it should be renamed.

Reply:
do you mean just that they shouldn't be so similarly named? or is there something else I'm missing?

Reply:
the name is too similar with w/W. Once the component is changed, lgtm

Reply:
I opted for `<NeverTrials/>`... but could also do `<TriallessExperiment/>` ? 🤔